docs/architecture/porting-existing-aspnet-apps/migration-considerations.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Migration considerations
+description: What does a team need to know in order to make the right decision about whether and how to migrate from ASP.NET MVC to .NET Core?
+author: ardalis
+ms.date: 11/13/2020
+---
+
+# Migration considerations
+
+The most fundamental question teams must answer when it comes to porting their apps to .NET Core is, should they port at all? In some cases, the best path forward may be to remain on the .NET Framework using ASP.NET MVC and/or Web API. In this chapter, we'll consider reasons why moving to .NET Core may make sense, and consider some scenarios and counterpoints for staying on .NET.
+
+## Is migration to .NET Core appropriate?
+
+Let's start with some of the reasons why you might want to move to .NET Core. There are quite a few, so don't consider this an exhaustive list.
+
+### Cross-platform support
+
+Apps built on .NET Core are truly cross-platform and can run on Windows as well as Linux and MacOS. This means not only can your developers use whatever hardware they want, but you'll also be able to host your application anywhere from local IIS to Azure in the cloud; from Linux Docker containers to IoT devices.
+
+### Performance and scalability
+
+Apps built with .NET Core are running on one of the fastest tech stacks available anywhere. If performance is a key concern for your app, you're almost certain to see a performance improvement by moving to .NET Core from ASP.NET MVC.
```

Comment:
Might make sense to call out some examples like Techempower or link to some case-studies here.

Reply:
Is there a press release by Microsoft somewhere I can reference? My quick search here came up empty - the regular TechEmpower site doesn't do a good job of showcasing .NET Core.

Reply:
thanks, updated.